newrelic-fluent-bit-output/fix-aarch64.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index f0aea08..cf86d20 100644
+--- a/Makefile
++++ b/Makefile
+@@ -6,7 +6,7 @@ linux/amd64:
+ 	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -buildmode=c-shared -o out_newrelic-linux-amd64-${VERSION}.so .
+ 
+ linux/arm64:
+-	CGO_ENABLED=1 GOOS=linux GOARCH=arm64 CC=aarch64-linux-gnu-gcc CXX=aarch64-linux-gnu-g++ go build -buildmode=c-shared -o out_newrelic-linux-arm64-${VERSION}.so .
++	CGO_ENABLED=1 GOOS=linux GOARCH=arm64 go build -buildmode=c-shared -o out_newrelic-linux-arm64-${VERSION}.so .
+ 
+ linux/arm/v7:
+ 	CGO_ENABLED=1 GOOS=linux GOARCH=arm CC=arm-linux-gnueabihf-gcc CXX=arm-linux-gnueabihf-g++ go build -buildmode=c-shared -o out_newrelic-linux-arm-${VERSION}.so .
```

Comment:
Do we want to do our future selves a favor and remove this for arm/v7 too?

Reply:
Yes, remove it for all of them tbh